pubpub/pubpub
# feat: don't show cookie banner except when using GA, add feature flag for never showing cookie banner, and don't load heap analytics

## Issue(s) Resolved
Remove heap, remove GDPR banner


## Test Plan

Keep cookies tab open

For this test, demo has google analytics set on dev, and jtrialerror also has GA set + has a feature flag set that prevents the banner from showing (this is a feature ELife needs)

Before each step, _clear your cookies_
____

1. Go to demo
2. Observe the analytics banner pop up
3. Click reject
4. Observe that no google cookies are set

____

1. Go to demo
2. Observe analytics banner pop up
3. Click accept
4. Observe that google cookies get set
5. Observe that a collect request gets sent

____

1. Go to demo
2. Log in
3. Go to /legal/settings
4. If enabled, disable
4.a. Observe that google cookies get removed
4.b. Navigate back to home page
4.c. Observe that neither banner pops up, nor that an event is sent to google
4.d. For some reason a google cookie is still set. I don't really know how to prevent this.
5. If disabled, enable
5.a. Observe that Google cookies get added and event is sent to Google

____

1. Go to jtrialerror
2. Observe no cookie banner
3. Observe no event sent to google

____

1. Go to jtrialerror
2. Log in
3. Go to /legal/settings
4. If enabled, disable
4.a. Observe that google cookies get removed
4.b. Navigate back to home page
4.c. Observe that neither banner pops up, nor that an event is sent to google
4.d. For some reason a google cookie is still set. I don't really know how to prevent this.
5. If disabled, enable
5.a. Observe that Google cookies get added and event is sent to Google

____

1. Go to any other community (HDSR or something)
2. Observe no cookie banner  popping up.

____


## Screenshots (if applicable)

## Optional

### Notes/Context/Gotchas

This PR does not yet change the wording on the cookie banner or on the legal page.

### Supporting Docs
